sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -56,103 +73,85 @@
   </v-navigation-drawer>
 </template>
 <script lang="ts">
-import { useRouter } from "vue-router";
+import { ref } from "vue";
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
 import { MenuModel, Role } from "@/types";
 import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
 
 export default {
   components: { CheckPermissionRole },
   setup() {
-    const router = useRouter();
+    const drawer = ref("drawer");
```

Comment:
Is there a reason to have this? If not please remove it. If it was copied from another code and there is no reason to keep it, please remove it.

Reply:
There is a purpose of v-model for the drawer as per vuetify. I think we should keep the v-model.

![image](https://user-images.githubusercontent.com/54600590/188211938-0681f298-91b8-41e5-964e-7be810a131c3.png)

Reply:
There is a purpose for every single property that exists in Vuetify 😄 
This one here does not seem to be used in the right way or causing any difference. Can you please provide a single benefit to keeping it? Maybe I am missing something.

Reply:
Understood after our discussion @andrewsignori-aot. @guru-aot the v-model is not useful for a permanent type drawer.

Reply:
Drawer is used to show the sidebar, without the property side bar is not showing.